### ISSUE TITLE
actions: build failed immediately

### ISSUE THREAD

**Issue by bnb** [nodejs/devcontainer#8]
It seems that today's build immediately crashed. I'm going to let it sit and see if tomorrow's build also crashes because the error... shouldn't have happened.

Error:
```
docker
buildx failed with: ERROR: failed to solve: process "/bin/sh -c /home/developer/scripts/install.sh && /home/developer/scripts/mkdir.sh && /home/developer/scripts/ccache.sh && /home/developer/scripts/clone.sh && /home/developer/scripts/build.sh && make install -C /home/developer/nodejs/node && /home/developer/scripts/ncu.sh" did not complete successfully: exit code: 100
```

**Comment by bnb** [comment]
It seems that the build has consistently failed for the past few runs. I’ll take some time tomorrow to poke around and see if I can get it working again.

**Comment by bnb** [comment]
cloned and ran build:

```
#8 16.71 Reading package lists...
#8 17.05 W: GPG error: https://cli.github.com/packages stable InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY C99B11DEB97541F0
#8 17.05 E: The repository 'https://cli.github.com/packages stable InRelease' is not signed.
------
executor failed running [/bin/sh -c /home/developer/scripts/install.sh && /home/developer/scripts/mkdir.sh && /home/developer/scripts/ccache.sh && /home/developer/scripts/clone.sh && /home/developer/scripts/build.sh && make install -C /home/developer/nodejs/node && /home/developer/scripts/ncu.sh]: exit code: 100
```

it's failing because of https://github.com/cli/cli/issues/6175

**Comment by bnb** [comment]
Seems like since that's been fixed it's been working continuously. Going to close this out 👍🏻
